controllers/apps/v1beta3/emqx_handler.go

```diff
@@ -602,7 +566,7 @@ func generateLoadedPlugins(instance appsv1beta3.Emqx, sts *appsv1.StatefulSet) (
 			Value: "/mounted/plugins/data/loaded_plugins",
 		},
 	)
-	sts.Spec.Template.Spec.Containers = []corev1.Container{container}
```

Comment:
Looks not make science, maybe need function for merge container by container name